packages/material-ui/src/Autocomplete/Autocomplete.test.js

```diff
@@ -580,19 +585,22 @@ describe('<Autocomplete />', () => {
       const handleSubmit = spy();
       const handleChange = spy();
       const { getAllByRole } = render(
-        <Autocomplete
-          disabledItemsFocusable
-          getOptionDisabled={(option) => option === 'two'}
+        <div
           onKeyDown={(event) => {
             if (!event.defaultPrevented && event.key === 'Enter') {
               handleSubmit();
             }
           }}
-          onChange={handleChange}
-          openOnFocus
-          options={['one', 'two', 'three']}
-          renderInput={(props2) => <TextField {...props2} autoFocus />}
-        />,
+        >
+          <Autocomplete
+            disabledItemsFocusable
+            getOptionDisabled={(option) => option === 'two'}
+            onChange={handleChange}
+            openOnFocus
+            options={['one', 'two', 'three']}
+            renderInput={(props2) => <TextField {...props2} autoFocus />}
+          />
+        </div>,
```

Comment:
These changes don't relate to the test description anymore.

Reply:
What do you mean? The tests assert that onSubmit and onChange aren't called. This is for testing the `should disable the option` part.

Reply:
> What do you mean?

What I wrote.

>  This is for testing the should disable the option part of the test description.

"disable the option" is not a sufficient description since it's nowhere defined what an "enabled option" does.

Reply:
Ok, I will try to come up with something better